README.md

```diff
@@ -1905,6 +1905,7 @@ Update Time, five active automations, webhooks.
 
 ## Other Free Resources
 
+  * [Wikimint Developer](https://developer.wikimint.com/p/tools.html) - Always free tools for web developers that includes CSS minify unminify, image optimizer, image resizer, case convertor, CSS validator, JavaScript compiler, HTML editor, etc.
```

Comment:
Site keeps popping up the consent page on every page no matter how many times you press ok. Doesn't seem ready for general use.

Reply:
Thanks for notifying the bug. Will fix it and inform you.

Reply:
Hi @ripienaar, I've fixed the consent issue. Can you please kindly review it?